src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/domain_name_management.html.twig

```diff
@@ -32,8 +32,8 @@
         <i class="material-icons">settings</i>
         {{ 'Manage domain name'|trans }}
       </h3>
-      <div class="card-block row">
-        <div class="card-text">
+      <div class="card-body">
+        <div class="form-wrapper">
```

Comment:
2 alternative to get rid of wrapper:
```suggestion
      <div class="card-body-form">
```
and then
```scss
.card-body-form {
    padding: 1.875rem $card-spacer-x+$spacer*0.5;
}
```

OR
```suggestion
      <div class="card-body form-wrapper">
```
with
```scss
.card-body.form-wrapper {
    padding: 1.875rem $card-spacer-x+$spacer*0.5;
}
```

Reply:
@davidglezz I would keep the forms separate from the card-body. The point was to keep the original boostrap syntax unhacked and only extend it. So people who come to Prestashop enviroment aren't suddenly wondering why XXX does something different than in default BS.